tools/appstreamcli.c

```diff
@@ -1089,7 +1089,7 @@ as_client_run_compose (const gchar *command, char **argv, int argc)
 	for (gint i = 2; i < argc; i++)
 		asc_argv[i-1] = argv[i];
 
-	return execv(ascompose_exe, (gchar * const*) asc_argv);
```

Comment:
This replacement is wrong, we must call `execv` here to replace the current process with the new one. If Windows doesn't have a compatibility interface for this, I'd rather like a separate codepath here for Windows that uses spawn_sync/posix_spawn and doesn't replace the parent (so the original behavior on non-Windows is preserved).